projects/storefrontlib/cms-components/cart/add-to-cart/add-to-cart.component.ts

```diff
@@ -83,6 +85,7 @@ export class AddToCartComponent implements OnInit, OnDestroy {
     protected currentProductService: CurrentProductService,
     protected cd: ChangeDetectorRef,
     protected activeCartService: ActiveCartService,
+    @Optional() protected addedToCartToastConfig?: AddedToCartToastConfig,
```

Comment:
I will suggest doing quickly `TODO(#13041): Remove deprecated constructors` to avoid conflicts.